spec/context_spec.rb

```diff
@@ -144,6 +144,15 @@
         expect(context.read('not_exists', 'fruit')).to be_nil
       end
     end
+
+    context 'case insensitive read' do
+      it "gets value no matter case of the header" do
+        context.instance_variable_get('@variables')[:headers] = OpenStruct.new(:case => 'insensitive')
+        expect(context.read('headers', 'case')).to eql("insensitive")
+        expect(context.read('headers', 'Case')).to eql("insensitive")
+      end
```

Comment:
@bchhay-splunk What happens if the user sends a symbol for the `object_key`? We should make sure the following expectations pass:
```suggestion
      it "gets value no matter case of the header" do
        context.instance_variable_get('@variables')[:headers] = OpenStruct.new(:case => 'insensitive')
        expect(context.read('headers', 'case')).to eql("insensitive")
        expect(context.read('headers', 'Case')).to eql("insensitive")
        expect(context.read(:headers, 'case')).to eql("insensitive")
        expect(context.read(:headers, 'Case')).to eql("insensitive")
      end
```

Reply:
nice thanks. i didn't have to make any real code changes to make this work, just needed to also add the symbol to the @case_insensitive_namespace. we'll have to keep that in mind in case we have to add other case-insensitive values. i'll be sure to make a comment of that when we use this new version of the gem in our runners.